src/components/SettingMenu/SettingMenu.tsx

```diff
@@ -0,0 +1,126 @@
+import React, { useRef, useState } from 'react';
+import { BsClockHistory } from 'react-icons/bs';
+import { FaEthereum } from 'react-icons/fa';
+import { HiOutlineDotsHorizontal } from 'react-icons/hi';
+import { IoIosLogOut } from 'react-icons/io';
+import { MdOutlineDashboardCustomize } from 'react-icons/md';
+import styled, { useTheme } from 'styled-components';
+import { useTransactionBuilderModal } from '../../hooks';
+import useOutsideAlerter from '../../hooks/useOutsideClick';
+import { Theme } from '../../utils/theme';
+import History from '../History';
+
+const MenuWrapper = styled.div`
+  z-index: 10;
+  position: absolute;
+  top: 40px;
+  right: 15px;
+  background: ${({ theme }) => theme.color.background.topMenu};
+  color: ${({ theme }) => theme.color.text.topMenu};
+  border-radius: 5px;
+  padding: 12px 16px;
+  font-size: 14px;
+  text-align: left;
+  box-shadow: rgb(0 0 0 / 24%) 0px 3px 8px;
+`;
+
+const MenuItem = styled.div`
+  display: flex;
+  justify-content: left;
+  align-items: center;
+  margin-bottom: 10px;
+  margin-right: 12px;
+  cursor: pointer;
+
+  a,
+  a:visited {
+    color: ${({ theme }) => theme.color.text.topMenu};
+    text-decoration: none;
+  }
+
+  a:hover {
+    color: ${({ theme }) => theme.color.text.settingsIcon};
+    text-decoration: none;
+  }
```

Comment:
I think we can create a separate component for the MenuItem anchor something like `MenuItemLink`